apps/src/code-studio/initApp/loadApp.js

```diff
@@ -26,6 +27,9 @@ import queryString from 'query-string';
 // Max milliseconds to wait for last attempt data from the server
 var LAST_ATTEMPT_TIMEOUT = 5000;
 
+const BLANK_SHARE_IMAGE_PATH = '/assets/blank_sharing_drawing.png';
```

Comment:
I wonder if we should [let webpack process this image](https://github.com/code-dot-org/code-dot-org/pull/12340) instead of referencing the relative URL like this. Also if something like `ARTIST_SHARE_FRAME` would be a better name - on my first read through I thought "why would we ever want a blank share image?"